tests/testutils/test_utils.go

```diff
@@ -404,3 +406,48 @@ func compress(content string) []byte {
 	_ = bw.Close()
 	return buf.Bytes()
 }
+
+func WithManager(t *testing.T, inst *instance.Instance) (err error, shouldRemoveUUID bool) {
```

Comment:
I don't see why you still return `shouldRemoveUUID`

Reply:
Because we might want to disable the Manager within a test and thus need it to call `DisableManager`.